src/Forms/ExportCustomTextFormat.cs

```diff
@@ -20,6 +19,8 @@ public ExportCustomTextFormat(string format)
             comboBoxNewLine.Items.Add(l.DoNotModify);
             comboBoxNewLine.Items.Add("||");
             comboBoxNewLine.Items.Add(" ");
+            comboBoxNewLine.Items.Add("{nl}{tab}");
+            comboBoxNewLine.Items.Add("{cr}{lf}");
 

```

Comment:
Hmm, why preset `{nl}{tab}` when it's useful only for my particular issue? Wouldn't it be better simply to list all the supported templates, just so that people are aware of them and can mix and match on their own? Like:

- `[Do not modify]`
- `||`
- ` `
- `{nl}`
- `{lf}`
- `{cr}{lf}` (this one's a combo to make it less likely for someone to use only `\r`)
- `{tab}`

And it might prevent some issues being opened in the future asking what `{nl}` is if you explicitly use `{environment newline}` instead.